### PR TITLE
[T3A3][T11-A1]Tan Hong Yu

### PR DESCRIPTION
**Pull request by hongyuhy**

### CODE REVIEW

**Comment by hongyuhy** [comment]
Ready for Review

File: src/seedu/addressbook/data/person/Address.java [hunk 1]
```diff
@@ -22,34 +27,40 @@
      */
     public Address(String address, boolean isPrivate) throws IllegalValueException {
         this.isPrivate = isPrivate;
-        if (!isValidAddress(address)) {
-            throw new IllegalValueException(MESSAGE_ADDRESS_CONSTRAINTS);
+
+        String[] splitUp = address.split(",");
+        if(splitUp.length != 4){
```
Comment (xdshang) [inline src/seedu/addressbook/data/person/Address.java]: Testing the length of `splitUp` is the simplest way to do validation. But in the future, if you need to do more sophisticated validation, you cannot write all those checking codes in one condition. So you still should use the method `isValidAddress(...)` here, and write the checking details in that method.

File: src/seedu/addressbook/data/person/PostalCode.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    public final String value;
+
+    public PostalCode(String postalcode){
+      
+        this.value = postalcode;
+    }
+
+    /**
+     * Returns true if a given string is a valid person email.
+     */
```
Comment (xdshang) [inline src/seedu/addressbook/data/person/PostalCode.java]: Remove these extra comments.

**Comment by xdshang** [comment]
@hongyuhy Some comments added. Please close the PR after reading comments.

**Pull request closed without merging**